onnx/backend/test/case/node/col2im.py

```diff
@@ -0,0 +1,344 @@
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+
+from ..base import Base
+from . import expect
+
+
+class Col2Im(Base):
+    @staticmethod
+    def export() -> None:
+        input = np.array(
+            [
+                [
+                    [1.0, 6.0, 11.0, 16.0, 21.0],  # (1, 5, 5)
+                    [2.0, 7.0, 12.0, 17.0, 22.0],
+                    [3.0, 8.0, 13.0, 18.0, 23.0],
+                    [4.0, 9.0, 14.0, 19.0, 24.0],
+                    [5.0, 0.0, 15.0, 20.0, 25.0],
+                ]
+            ]
+        ).astype(np.float32)
+
+        image_shape = np.array([5, 5]).astype(np.int64)
+        block_shape = np.array([1, 5]).astype(np.int64)
+        node = onnx.helper.make_node(
+            "Col2Im", ["input", "image_shape", "block_shape"], ["output"]
+        )
+
+        output = np.array(
```

Comment:
I thought you mentioned you had a python implementation of the op? If so, it would be helpful to include that here.

Reply:
It is a Github project https://github.com/f-dangel/unfoldNd/